test/ui/starter-select.test.ts

```diff
@@ -545,7 +545,7 @@ describe("UI - Starter select", () => {
     let starterSelectUiHandler: StarterSelectUiHandler | undefined;
     await new Promise<void>((resolve) => {
       game.onNextPrompt("SelectStarterPhase", UiMode.STARTER_SELECT, () => {
-        starterSelectUiHandler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+        starterSelectUiHandler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        starterSelectUiHandler = game.scene.ui.getCurrentHandler<StarterSelectUiHandler>();
```